build.gradle

```diff
@@ -506,7 +506,7 @@ integTest {
             systemProperty "tests.cluster.${cluster.name}.http_hosts", "${-> allHttpSocketURI}"
             systemProperty "tests.cluster.${cluster.name}.transport_hosts", "${-> alltransportSocketURI}"
             systemProperty "tests.cluster.${cluster.name}.security_enabled", "${-> securityEnabled.toString()}"
-            systemProperty "tests.cluster.${cluster.name}.total_nodes", "${-> _numNodes.toString()}"
```

Comment:
Why do we need to change the property name for consume?

Reply:
I created a unified function to check if multiNodeConfiguration is present

Reply:
Let's keep the same key as before. `isMultiNodeClusterConfiguration` can take a cluster name as argument.

Reply:
Done